pages/deployment/aws.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: Deploy Memgraph on AWS EC2 instance 
+description: Learn how to deploy your Memgraph instance on the AWS EC2 instance.
+---
+
+import { Card, Cards, Tabs, Tab } from 'nextra/components'
+
+# Deploy Memgraph on AWS EC2 instance
+
+This guide will show you how to deploy Memgraph on an AWS EC2 instance. The
+guide will cover only the specific bits that are different from the general
+deployment guide you can find for native [Linux](./linux.mdx) or
+[Docker](./docker.mdx) deployments.
+
+This guide assumes you have an AWS account and are familiar with the AWS
+console. If you are not, you can follow the [AWS EC2
+documentation](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EC2_GetStarted.html).
+
+
+## Creating a new EC2 instance 
+
+The first step is to have an EC2 instance running on AWS, where you will deploy
+Memgraph. Here are some guidelines for opening a new EC2 instance for Memgraph. 
+
+### Picking the OS 
+
+When creating the EC2 instance, you need to pick the OS you wish to have on the
+instance. If you are going to use the Memgraph native installation, you need to
+pick a Linux distribution that Memgraph supports.
+
+Memgraph supports multiple Linux distributions. Memgraph packages for supported distributions and versions can be downloaded 
+from the [following page](/getting-started/install-memgraph/direct-download-links#linux)
+
+
+This way, you will manage the native installation of Memgraph, which can bring
+some of the speed benefits. 
+
+If you are going to use the Memgraph Docker image, you can pick any Linux
+distribution you prefer, as long as you can run Docker on it.
+
```

Comment:
We don't mention anything about choosing an architecture (x86 or arm). Please add that.